src/components/AdminPageTable/AdminPageHeader.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/jsx-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Thead, Tr, Th, Checkbox, Button } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Checkbox' is defined but never used.